pkg/agent/controller/egress/ip_assign_advertise.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package egress
+
+type INodeEgressAssigner interface {
```

Comment:
Is it common to add "I" for interface?
I would suggest to simplify the interfaces to:
```
type EgressIPAssigner interface {
	AssignEgressIP(egressIP string) error
	UnAssignEgressIP(egressIP string) error
}
```

Reply:
It is the most popular to name interfaces with suffix "er". I think it's also common naming an interface with prefix “I”, or suffix “Interface”.